lib/membrane/video_compositor/request.ex

```diff
@@ -0,0 +1,122 @@
+defmodule Membrane.VideoCompositor.Request do
+  @moduledoc false
+
+  require Membrane.Logger
+
+  alias Membrane.VideoCompositor
+  alias Membrane.VideoCompositor.OutputOptions
+
+  @local_host_url "127.0.0.1"
+  @type request_result ::
+          {:ok, Req.Response.t()}
+          | {:error_response_code, Req.Response.t()}
+          | {:error, Exception.t()}
+
+  @spec init(non_neg_integer(), Membrane.Time.t(), boolean(), :inet.port_number()) ::
+          request_result()
+  def init(framerate, stream_fallback_timeout, init_web_renderer?, vc_port) do
+    %{
+      type: "init",
```

Comment:
not sure if it's idiomatic in elixir, but wouldn't field like type be a good candidate to use atoms instead of strings?